ephemeris/generate_tool_list_from_ga_workflow_files.py

```diff
@@ -49,14 +48,24 @@ def translate_workflow_dictionary_to_tool_list(tool_dictionary, panel_label):
     tool_list = []
     for tool in starting_tool_list:
         sub_dic = {'name': tool['name'], 'owner': tool['owner'], 'revision': tool['changeset_revision'],
-                   'tool_panel_section_label': panel_label, 'tool_shed_url': 'https://'+tool['tool_shed']}
+                  'tool_panel_section_label': panel_label, 'tool_shed_url': 'https://'+tool['tool_shed'],
+                  'zinstall_resolver_dependencies': 'True'}
```

Comment:
Yes this is the nasty trick I mentionned... This is not a bug, this is a feature :-/